aplib.net-demo/Assets/Scripts/Doors/DoorCollision.cs

```diff
@@ -0,0 +1,24 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+/// <summary>
+/// This class handles collisions with the player and makes sure that the parent object (the door) disappears/opens when the players is in range and the prerequisites are met (the right key, or in this case the right player ID)
+/// Set the ID in the editor for the collider
+/// </summary>
+public class DoorCollision : MonoBehaviour
+{
+    public int doorID;
+    [SerializeField] private GameObject parent;
+    /// <summary>
+    /// Checks if the player has the right ID, and destroys the door if true
+    /// </summary>
+    /// <param name="other">the object that collides with the collider</param>
```

Comment:
Here other needs to be collidingObject as this has probably been changed. Change it to that if you would want to have the tooltip.